src/app/products/products.module.ts

```diff
@@ -72,7 +75,12 @@ import { ViewTaxComponentComponent } from './manage-tax-components/view-tax-comp
   imports: [
     SharedModule,
     ProductsRoutingModule,
-    PipesModule
+    PipesModule,
```

Comment:
importing shared module is enough, angular material doesn't needs to be separately imported, import common module instead of Pipes module